src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -1,11 +1,13 @@
-(ns status-im.contexts.wallet.send.flow-config)
+(ns status-im.contexts.wallet.send.flow-config
+  (:require
+    [status-im.contexts.wallet.send.utils :as send-utils]))
 
 (defn- collectible-selected?
   [db]
   (let [collectible-stored (-> db :wallet :ui :send :collectible)
         tx-type            (-> db :wallet :ui :send :tx-type)]
     (and (some? collectible-stored)
-         (= tx-type :collectible))))
+         (send-utils/is-collectible? tx-type))))
```

Comment:
Maybe I'm over thinking this, but do we really want to import a function to check this?

The `=` call before is clear enough and simpler since. Also the function name `is-collectible` isn't clear enough, IMO it suggests if an entity (e.g. a map) is a collectible.

cc: @ilmotta wdyt about this practice?

Reply:
Good question @ulisesmac. I'll just share some thoughts, but I don't see much of a problem with the current change. Up to you guys really.

- It's common practice to isolate certain business rules in functions, even if they're very simple. The advantage of having a separate function like `is-collectible?` is that if the logic to detect a collectible transaction type changes, we'll only need to change the implementation detail, not hunt all places doing equality comparisons. It's a trade-off.
- Some clojurists would try to build systems using more namespaced keywords. For instance, `:collectible` refers to what? `:tx/collectible` gives a different clarity. This to me is the curse also of the tight integration we have with status-go, because we tend to consume values directly from there and we can't properly encode them for idiomatic Clojure due to performance...
- `is-collectible?` is clear to me, but only if I read using the argument `tx-type`. The `is-` prefix is non-idiomatic. Suggestion to rename: `send-utils/tx-type-collectible?`, this way if, in the future, this function is used in places where the argument name is not that clear, the function name will be self-explanatory.
- The implementation of `is-collectible?` could avoid reconstructing the set on every call.

Reply:
Thanks for the feedback @ilmotta 

I agree with the rename suggestion and I wasn't aware of the checks inside the helper function.